src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IUnconfiguredProjectVsServices.cs

```diff
@@ -20,5 +20,10 @@ internal interface IUnconfiguredProjectVsServices : IUnconfiguredProjectCommonSe
         ///     Gets <see cref="IVsProject4"/> provided by the <see cref="UnconfiguredProject"/>.
         /// </summary>
         IVsProject4 VsProject { get; }
+
+        /// <summary>
+        ///     Gets <see cref="IPhysicalProjectTree"/> provided by the <see cref="UnconfiguredProject"/>.
+        /// </summary>
+        IPhysicalProjectTree ProjectTree { get; }
```

Comment:
Why in the VS layer?

Reply:
I don't understand. This member existed on the implementation. This just pulls it up to the interface. It's needed in `DependenciesTreeSearchProvider`.